tests/schemas/api_v3/transactionsDryRun.schema.js

```diff
@@ -20,7 +20,7 @@ const regex = require('./regex');
 const TRANSACTION_VERIFY_RESULT = {
```

Comment:
To avoid duplication, instead of re-defining it here, can we please import the original implementation from the microservice?